client/webpack.common.config.js

```diff
@@ -6,12 +6,8 @@ module.exports = {
 
   // the project dir
   context: __dirname,
-  entry: ['./assets/javascripts/App'],
+  entry: ['jquery', 'jquery-ujs', './assets/javascripts/App'],
```

Comment:
[Documentation](http://webpack.github.io/docs/configuration.html#entry) says that Object instead of Array or String is used to get multiple outputs. I'm not sure this will work for one output.